jobs/jobs-operator/manifests/overlays/test/kustomization.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: kopf
+resources:
+- ../../base
+- curibio-operators-creds.yaml
+
+patchesJson6902:
+- target:
+    kind: Deployment
+    name: operators
+  patch: |-
+    - op: replace
+      path: /spec/template/spec/containers/0/image
+      value: "077346344852.dkr.ecr.us-east-2.amazonaws.com/jobs-operator:0.0.1"
```

Comment:
did you terraform the new ecr repos for this image and the queue-processor image?

Reply:
True, I didn't do that, will do it now.

Reply:
@jperson they're good to go now!